src/tooltip/tooltip.ts

```diff
@@ -53,15 +55,16 @@ export class NgbTooltip implements OnInit, OnDestroy {
    */
   @Input() container: string;
   /**
- * Emits an event when the tooltip is shown
- */
+   * Emits an event when the tooltip is shown
+   */
   @Output() shown = new EventEmitter();
   /**
    * Emits an event when the tooltip is hidden
    */
   @Output() hidden = new EventEmitter();
 
   private _ngbTooltip: string | TemplateRef<any>;
+  private _ngbTooltipWindowId = `tooltip-${nextId++}`;
```

Comment:
This should probably be made a little more specific to ng-bootstrap, i.e. `ngb-tooltip-${nextId++}`